package.json

```diff
@@ -116,7 +116,7 @@
     "sift": "^3.2.7",
     "style-loader": "^0.16.1",
     "superagent": "^3.5.2",
-    "svgo": "^0.7.2",
+    "svg-url-loader": "^2.0.2",
```

Comment:
Checked the output, and it's already smaller than optimized svgo version, because of the base64. We have just a few SVGs so I wouldn't block on that